community/organizations.md

```diff
@@ -0,0 +1,115 @@
+## Julia GitHub Groups
+
+The following is a non-comprehensive list of Julia GitHub groups grouped by domain:
+
+### Julia
+
+@@tight-list
+* [JuliaLang](https://github.com/JuliaLang) - A fresh approach to numerical computing
+* [JuliaRegistries](https://github.com/JuliaRegistries)
+* [JuliaCon](https://github.com/JuliaCon) - The Julia Conference
+* [JuliaAcademy](https://github.com/JuliaAcademy) - The definitive source for learning all things Julia.
+* [JuliaCommunity](https://github.com/JuliaCommunity) - Organizing Julia Community Initiatives
+@@
+
+### General
+
+@@tight-list
+* [JuliaDocs](https://github.com/juliadocs) – Documentation-related packages for Julia ([Gitter](https://gitter.im/juliadocs/users))
```

Comment:
Are you still monitoring Gitter @mortenpi?

Reply:
I think I still get email notifications, but I wouldn't say that it's the preferred way of getting support on Documenter or JuliaDocs. #documentation in Slack or Zulip, or Discourse would be better.